### ISSUE TITLE
standard:indent cannot format multiline where clauses with tab indentation

### ISSUE THREAD

**Issue by autaut03** [pinterest/ktlint#3362]
Hi! ktlint 1.8.0 appears unable to produce valid formatting for multiline `where` clauses when tabs are enabled.

`.editorconfig`:

```editorconfig
root = true

[*.{kt,kts}]
indent_style = tab
indent_size = 4
```

`Repro.kt`:

```kotlin
private interface First

private interface Second

private fun <T> example(value: T)
    where T : First,
          T : Second {
    println(value)
}
```

Run:

```shell
ktlint --format Repro.kt
ktlint Repro.kt
```

The formatter correctly changes the function body indentation to a tab, but leaves the `where` clause indented with spaces. The following lint run then reports:

```text
Repro.kt:6:1: Unexpected space character(s) (standard:indent)
Repro.kt:7:1: Unexpected space character(s) (standard:indent)
```

`--format` also warns that it could not resolve all autocorrectable violations, and rerunning it does not help.

Expected: `ktlint --format` should produce output accepted by a subsequent ktlint check when `indent_style = tab`.

This may be related to #2175, but that report used space indentation and concerned alignment width; this reproduction is specific to tab indentation.

Environment: standalone ktlint CLI 1.8.0 on macOS.
